packages/backend-defaults/config.d.ts

```diff
@@ -420,6 +420,16 @@ export interface Config {
        */
       connection:
         | string
+        | {
+            /**
+             * The specific config for Azure database for PostgreSQL with Entra authentication
+             */
+            type: 'azure';
+            /**
+             * How many milliseconds before token expiration to refresh the token Defaults to 180,000 (3 minutes)
+             */
+            allowedClockSkewMs?: number;
```

Comment:
Maybe you can use `Duration | HumanDuration` here instead similarly to how this is done in the [SchedulerServiceTaskScheduleDefinition](https://github.com/backstage/backstage/blob/d651ce74623869a27964472644969766d0e9fbb9/packages/backend-plugin-api/src/services/definitions/SchedulerService.ts#L107C1-L107C37)
```suggestion
            allowedClockSkew?: Duration | HumanDuration;
```

Then user could write for example:
```yaml
...
connection:
  type: 'azure'
  allowedClockSkew: 
    minutes: 10
...
```

Reply:
I didn't notice that `HumanDuration` type before. That looks like a good improvement for the configuration. I see there are two other ways `HumanDuration` is used in this `Config` interface. One uses `string | HumanDuration` and the other uses `number | HumanDuration | string`. And I see that the implementation code in both use the `readDurationFromConfig` function. I think I'll go with the `string | HumanDuration` approach for the config.

Reply:
I only noticed it the other day myself. What would the `string` look like? Something like though `00:05:00`?

Reply:
Correct, string and HumanDuration.

If a string, it gets parsed by the `ms` library, which supports human text forms like "20ms" or "1 minute" etc